doc/nrf/security/tfm.rst

```diff
@@ -158,8 +157,8 @@ By default, one of the ports is used by the non-secure UART0 peripheral from the
 There are several options to get UART output from the secure TF-M:
 
 * Disable the output for the network core and change the pins used by TF-M.
-  The network core will usually have an |NCS| child image.
-  To configure a child image, see Configuration of the child image section described in :ref:`ug_nrf5340_multi_image`.
+  The network core will usually have an |NCS| image.
+  To configure an image, see the Configuration namespacing section in :ref:`sysbuild`.
```

Comment:
Ideally, we should link to the NCS's ``configuration_system_overview_sysbuild``, but in this case it makes sense to link to Zephyr's docs.
```suggestion
  To configure an image, see the Configuration namespacing section in Zephyr's documentation about :ref:`sysbuild`.
```